src/Microsoft.Framework.Runtime.Hosting/DependencyProviders/GlobalSettings.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Framework.Runtime.Hosting.DependencyProviders
+{
+    public class GlobalSettings
```

Comment:
Done. I'm actually going to have the new runtime remove support for "sources", since that is in the plan anyway. Thoughts?